util/stopwaiter/stopwaiter.go

```diff
@@ -73,9 +87,36 @@ func (s *StopWaiterSafe) StopOnly() {
 }
 
 // Stopping multiple times, even before start, will work
-func (s *StopWaiterSafe) StopAndWait() {
+func (s *StopWaiterSafe) StopAndWait() error {
+	return s.stopAndWaitImpl(stopDelayWarningTimeout)
+}
+
+func getAllStackTraces() string {
+	buf := make([]byte, 64*1024*1024)
+	size := runtime.Stack(buf, true)
+	builder := strings.Builder{}
+	builder.Write(buf[0:size])
+	return builder.String()
```

Comment:
@magicxyyz it's not worth fixing but for the future you can just cast a byte slice to a string, e.g. `return string(buf[0:size])`